app/models/ability.rb

```diff
@@ -0,0 +1,71 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new
+
+    can :manage, :all if user.is_admin?
+    can [:manage_item, :manage_content, :manage_desc_metadata, :register_apo], :all if user.is_manager?
+    can [:view_item, :view_metadata, :view_content], :all if user.is_viewer?
```

Comment:
Yes, permissions cascade as expected, and short-circuit when it gets an affirmative response.

Agreed, the difference between the application `admin` and `manager` roles is very unclear. Maybe @LynnMcRae can shed some light.